xmodule/video_block/video_handlers.py

```diff
@@ -527,29 +541,21 @@ def _studio_transcript_upload(self, request):
             new_language_code = request.POST['new_language_code']
             transcript_file = request.POST['file'].file
 
-            if not edx_video_id:
-                # Back-populate the video ID for an external video.
-                # pylint: disable=attribute-defined-outside-init
-                self.edx_video_id = edx_video_id = create_external_video(display_name='external video')
+            isLibrary = isinstance(self.usage_key.context_key, LibraryLocatorV2)
```

Comment:
Nit: python vars should be `is_library` not `isLibrary`

Reply:
Updated https://github.com/openedx/edx-platform/pull/36058/commits/6903be2b674410d06f07572f7acc6c37b3f23482